<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDialog</class>
 <widget class="QDialog" name="SelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QLabel" name="btn_close">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>4</y>
      <width>26</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>4</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Select City</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>508</width>
     <height>182</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabAbout">
    <attribute name="title">
     <string>Chinese Cities</string>
    </attribute>
    <widget class="QComboBox" name="county_combo">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="city_combo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="province_combo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitBtn">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>100</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabCopyright">
    <attribute name="title">
     <string>Foreign Cities</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>13</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please enter a city name to search&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>53</y>
       <width>70</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>52</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>100</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okBtn</tabstop>
  <tabstop>quitBtn</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>searchBtn</tabstop>
  <tabstop>province_combo</tabstop>
  <tabstop>city_combo</tabstop>
  <tabstop>county_combo</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
